x-pack/test/rule_registry/common/lib/helpers/cleanup_target_indices.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { GetService } from '../../types';
+import { User } from '../authentication/types';
+import { getAlertsTargetIndices } from './get_alerts_target_indices';
+
+export const cleanupTargetIndices = async (getService: GetService, user: User, spaceId: string) => {
+  const es = getService('es');
+  const { body: targetIndices } = await getAlertsTargetIndices(getService, user, spaceId);
+  try {
+    const aliasMap = await es.indices.getAlias({
+      name: targetIndices,
+      allow_no_indices: true,
+      expand_wildcards: 'open',
+    });
+    const indices = Object.keys(aliasMap);
+    expect(indices.length > 0).to.be(true);
+    return es.indices.delete({ index: indices });
+  } catch (err) {
+    // do nothing
```

Comment:
`console.warn` maybe?

Reply:
Actually we should probably just ignore any 404 or missing index errors and throw for everything else.